src/bindings/c/tests/ov_compiled_model_test.cpp

```diff
@@ -135,6 +135,10 @@ TEST_P(ov_compiled_model_test, get_property) {
     OV_EXPECT_OK(ov_core_compile_model(core, model, device_name.c_str(), 0, &compiled_model));
     EXPECT_NE(nullptr, compiled_model);
 
+    const char* key = ov_property_key_auto_batch_timeout;
+    OV_EXPECT_OK(ov_compiled_model_set_property(compiled_model, key));
+    ov_free(key);
```

Comment:
Could we also add test of get_property() for `ov_property_key_auto_batch_timeout` ?